data/clients.json

```diff
@@ -683,6 +685,14 @@
 			"url": "https://github.com/GeorgeSchiro/GetCert",
 			"category": "Windows / IIS",
 			"comments": "(simple GUI - .Net, C#, WPF, WCF)"
+		},
+		{
+			"name": "acme-lw-d",
+			"url": "https://github.com/cschlote/acme-lw-d.git",
```

Comment:
Why `https://github.com/cschlote/acme-lw-d.git` and not simply `https://github.com/cschlote/acme-lw-d`?

Reply:
That's the URL, which copied from the clone button on GitHub. Maybe it will work without the .git suffix as well.

Reply:
Both https://github.com/cschlote/acme-lw-d.git and https://github.com/cschlote/acme-lw-d work for cloning, and also for visiting the repo via the web. However, when visiting https://github.com/cschlote/acme-lw-d.git via web, you get redirected to https://github.com/cschlote/acme-lw-d, suggesting that's more canonical for web use. So I'd recommend changing this to https://github.com/cschlote/acme-lw-d, since the purpose of this list is for people clicking through on the web.

Reply:
Good point. Changed code.